src/go/rpk/pkg/cli/cmd/topic/consume_test.go

```diff
@@ -164,83 +165,113 @@ func TestParseConsumeTimestamp(t *testing.T) {
 	for i, test := range []struct {
 		in string
 
-		length int
-		at     time.Time
-		end    bool
-		expErr bool
+		length  int
+		startAt time.Time
+		endAt   time.Time
+		end     bool
+		expErr  bool
 	}{
-		{
+		{ // 0
 			in:     "",
 			expErr: true,
 		},
 
-		{
-			in:     "1648021901749:asdf",
```

Comment:
What happened to the asdf cases here?

Reply:
This just fails with 
```
invalid --offset "@2023-01-05T09:00:00Z:asdf": unable to parse duration in "asdf"
```
Do we need this to be tested?

Reply:
Not critically important, but they were negative tests: ensuring things fail when they should.

Reply:
fixed in bf66adf